src/components/learner-credit-management/tests/MultipleBudgetsPage.test.jsx

```diff
@@ -84,6 +84,15 @@ jest.mock('../../EnterpriseSubsidiesContext/data/hooks', () => ({
           isCurrent: true,
           isRetired: false,
         },
+        {
+          source: 'subsidy',
+          id: '392f1fe1-ee91-4f44-b174-13ecf59866ef',
+          name: 'Subsidy 6 expiring',
+          start: '2020-06-07T15:38:29Z',
+          end: '2025-04-07T15:38:30Z',
```

Comment:
will this end date cause the test to start failing in April? If so, can we dynamically generate this end date instead?